cas-server-core-api/src/main/java/org/jasig/cas/services/RegisteredService.java

```diff
@@ -179,5 +179,11 @@
      * @since 4.1
      */
     RegisteredServicePublicKey getPublicKey();
-    
+
+    /**
+     * Whether we should only keep the latest service ticket for this service to save memory.
+     *
+     * @return whether we should only keep the latest service ticket for this service
+     */
+    boolean isOnlyKeepLatestST();
```

Comment:
I think it would be easier for the adopter, if you changes this option to "onlyTrackMostRecentSession".